index.html

```diff
@@ -61,13 +61,204 @@ <h1>Hey there. I'm Jos Kalenda <br>
 
       </div>
     <div class="arrow--icons">
-  
       <a href="#">
         <img src="./assets/icons/arrow-down.png" alt="scrolling">
       </a>
     </div>
     </section>
+    <section class="recent--work--section">
+      <div class="recent--work--title">
+        <h1>My Recent Works</h1>
+        <img src="./assets/back-img/Indicator.svg" alt="line-indicator">
+      </div>
+      <ul class="main--works--section">
+        <li class="card--section">
+          <div class="image--container">
+            <a href="#"></a>
+          </div>
+          <div class="btn--container">
+            <h4>Multi-Post Stories Gain+Glory</h4>
+            <button class="btn" type="button">Ruby on rails</button>
+            <button class="btn" type="button">css</button>
+            <button class="btn" type="button">JavaScript</button>
+            <button class="btn" type="button">html</button>
+          </div>
+          <div class="see--project">
+            <button class="btn--style" type="button">See Project</button>
+          </div>
+        </li>
+        <li class="card--section">
+          <div class="image--container">
+            <a href="#"></a>
+          </div>
+          <div class="btn--container">
+            <h4>Multi-Post Stories Gain+Glory</h4>
+            <button class="btn" type="button">Ruby on rails</button>
+            <button class="btn" type="button">css</button>
+            <button class="btn" type="button">JavaScript</button>
+            <button class="btn" type="button">html</button>
+          </div>
+          <div class="see--project">
+            <button class="btn--style" type="button">See Project</button>
+          </div>
+        </li>
+        <li class="card--section">
+          <div class="image--container">
+            <a href="#"></a>
+          </div>
+          <div class="btn--container">
+            <h4>Multi-Post Stories Gain+Glory</h4>
+            <button class="btn" type="button">Ruby on rails</button>
+            <button class="btn" type="button">css</button>
+            <button class="btn" type="button">JavaScript</button>
+            <button class="btn" type="button">html</button>
```

Comment:
Same thing here
- [ ] On of the requirements says that the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `<ul>` and each of them a `<li>`. Just like this :point_down: 

```
 <ul class="techno-list">
      <li> <button class="btn" type="button">Ruby on rails</button></li>
      <li><button class="btn" type="button">css</button></li>
      <li><button class="btn" type="button">JavaScript</button></li>
      <li><button class="btn" type="button">html</button></li>
</ul>
```